bleachbit/Action.py

```diff
@@ -408,15 +409,13 @@ def run_process():
                     rc = 0  # unknown because we don't wait
                     from subprocess import Popen
                     Popen(self.cmd)
-            except Exception, e:
+            except Exception as e:
                 raise RuntimeError(
                     'Exception in external command\nCommand: %s\nError: %s' % (self.cmd, str(e)))
             else:
                 if not 0 == rc:
-                    logger = logging.getLogger(__name__)
```

Comment:
With the `getLogger` we can get separate loggers for `bleachbit.foo` and `bleachbit.bar` and collect logs for both with a logger for just `bleachbit`. Currently we don't do this but I'll revert the `getLogger` removals just in case we might want to some day.